langchain4j-weaviate/src/test/java/dev/langchain4j/store/embedding/weaviate/LocalWeaviateNoMetadataEmbeddingStoreIT.java

```diff
@@ -0,0 +1,47 @@
+package dev.langchain4j.store.embedding.weaviate;
+
+import dev.langchain4j.data.segment.TextSegment;
+import static dev.langchain4j.internal.Utils.generateUUIDFrom;
+import dev.langchain4j.model.embedding.AllMiniLmL6V2QuantizedEmbeddingModel;
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.store.embedding.EmbeddingStore;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.weaviate.WeaviateContainer;
+
+
+import dev.langchain4j.store.embedding.EmbeddingStoreWithoutMetadataIT;
+
+@Testcontainers
+class LocalWeaviateNoMetadataEmbeddingStoreIT extends EmbeddingStoreWithoutMetadataIT {
+
+    @Container
+    static WeaviateContainer weaviate = new WeaviateContainer("semitechnologies/weaviate:latest")
+            .withEnv("QUERY_DEFAULTS_LIMIT", "25")
+            .withEnv("DEFAULT_VECTORIZER_MODULE", "none")
+            .withEnv("CLUSTER_HOSTNAME", "node-nometadata");
+
+    private final EmbeddingStore<TextSegment> embeddingStore = WeaviateEmbeddingStore.builder()
+            .scheme("http")
+            .host(weaviate.getHost())
+            .port(weaviate.getFirstMappedPort())
+            .objectClass("Test" + generateUUIDFrom("LocalWeaviateNoMetadataEmbeddingStoreIT").replace("-", ""))
```

Comment:
```suggestion
            .objectClass("Test" + randomUUID().replace("-", ""))
```
Tests fail otherwise